.changelog/20250717110708_ck_18842_changelog_pre_release.md

```diff
@@ -0,0 +1,9 @@
+---
+type: Feature
+scope:
+  - ckeditor5-dev-changelog
+closes:
+  - https://github.com/ckeditor/ckeditor5/issues/18842
+---
+
+Introduced support for non-stable releases in the changelog generating tool.
```

Comment:
In the issue (https://github.com/ckeditor/ckeditor5/issues/18842), we described the entire process of how it works. Could we be more specific and explain what "Introduced support" means?

Reply:
Updated description in [be7ac14](https://github.com/ckeditor/ckeditor5-dev/pull/1183/commits/be7ac140d67cb518bc265fed7dd442aa69b93246).